cmd/root.go

```diff
@@ -229,6 +231,8 @@ func init() {
 	viper.SetDefault("log-level", int(logrus.InfoLevel))
 	viper.BindPFlag("log-file", rootCmd.Flags().Lookup("log-file"))
 	viper.SetDefault("log-file", "./server.log")
+	viper.BindPFlag("verus-conf-path", rootCmd.Flags().Lookup("verus-conf-path"))
+	viper.SetDefault("verus-conf-path", "./VRSC.conf")
 	viper.BindPFlag("zcash-conf-path", rootCmd.Flags().Lookup("zcash-conf-path"))
```

Comment:
zcash-conf-path is obsolete, please remove it.

Reply:
Moved away from .conf files altogether. We now pass the verusd chain name, URL, username and password in as command line options.